lldb/source/Host/posix/MainLoopPosix.cpp

```diff
@@ -159,6 +160,22 @@ MainLoopPosix::RunImpl::RunImpl(MainLoopPosix &loop) : loop(loop) {
   read_fds.reserve(loop.m_read_fds.size());
 }
 
+int MainLoopPosix::RunImpl::StartPoll(
+    std::optional<MainLoopPosix::TimePoint> point) {
+#if HAVE_PPOLL
```

Comment:
I noticed that HAVE_PPOLL is always defined but only its value will either be 0 or 1, so added accordingly.